lib/constants.js

```diff
@@ -180,4 +180,88 @@ exports.countryNameMap = {
   'Dominican Republic': 'DO',
   'Burkina Faso': 'BF',
   Uzbekistan: 'UZ',
-};
\ No newline at end of file
+};
+
+exports.helpContent = `
+
+/$$   /$$ /$$$$$$$$ /$$       /$$$$$$$
+| $$  | $$| $$_____/| $$      | $$____$$
+| $$  | $$| $$      | $$      | $$    $$
+| $$$$$$$$| $$$$$   | $$      | $$$$$$$/
+| $$__  $$| $$__/   | $$      | $$____/
+| $$  | $$| $$      | $$      | $$
+| $$  | $$| $$$$$$$$| $$$$$$$$| $$
+|__/  |__/|________/|________/|__/
+
+---------------------------------------------------------------------------------
+
+# Source 1 stats - updated once a day from John Hopkins University
+https://corona-stats.online
+
+---------------------------------------------------------------------------------
+
+# Source 2 stats - updated every 15 minutes from worldometers.info
+https://corona-stats.online?source=2
+
+---------------------------------------------------------------------------------
+
+# Country wise stats
+
+## Format:
+https://corona-stats.online/<countryCode>
```

Comment:
It seems you need to escape `<` and `>` or something @sagarkarira here and in lines 267-258

Reply:
I was just conveying the format. It is not an actual API. What better can be done here to convey?

Reply:
You need to escape the characters otherwise they don't show up in the resulting HTML :)

Reply:
Got it. You are right. I will fix it.